packages/backend/jobs-manager/service/src/modules/database/subscriptions/subscriptions.dto.ts

```diff
@@ -27,5 +27,11 @@ export class PatchSubscriptionDto {
   @Equals(true)
   @IsBoolean()
   @Type(() => Boolean)
```

Comment:
Should be removed since it is not a GET request, the boolean should not be coming in as a string. In the GET requests, all the values are string since they are the equivalent of the content-type `application/x-www-form-urlencoded`.

```suggestion
```